src/win/process.c

```diff
@@ -367,12 +369,14 @@ static WCHAR* search_path(const WCHAR *file,
   name_has_ext = (dot != NULL && dot[1] != L'\0');
```

Comment:
Suggestion: move this into the `else` block below. That's now the the only place that uses it.